main/TPS546.c

```diff
@@ -0,0 +1,748 @@
+#include "driver/i2c.h"
+#include "esp_log.h"
+#include <stdio.h>
+#include <stdint.h>
+#include <math.h>
+#include <string.h>
+
+#include "pmbus_commands.h"
+#include "TPS546.h"
+
+#define I2C_MASTER_SCL_IO 48 /*!< GPIO number used for I2C master clock */
```

Comment:
on all these defines, only the I2C_MASTER_NUM should be needed for you here, remove the others